### PR TITLE
Road to arm

### PR DESCRIPTION
**Pull request by Maleware**
Larger article about how to produce multi-arch images with docker and cargo. It's meant to be a knowledge collection with a little good sounding framework. 

Hopefully it's fun to read

### CODE REVIEW

File: modules/concepts/pages/road-to-multiarch.adoc [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+However, we already tested those changes within the link:https://github.com/stackabletech/kafka-operator/compare/main...stackable-experimental[Kafka Operator]. We plan to have more branches called *stackable-experimental*. 
+
+=== Target State
+
+As a wishful state in general, we would like to merge the stackable-experimental repository with the stackable repository. From there it should be a standard to provide multi-arch images to all products and operators. This means, all merged PR's are available to (currently) both platforms. 
+
+==== Products
+
+Our products have as shown above an overall desireable state. Only thing left to do is to test it in a broughter fashion and merge things into the stackable repository.
```
Comment (maltesander) [inline modules/concepts/pages/road-to-multiarch.adoc:239]:
```suggestion
As shown above, our products have an overall desirable state. The only thing left is test it in a broader fashion and merge things into the stable stackable repository to be accessible for everyone. 
```

File: modules/concepts/pages/road-to-multiarch.adoc [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+As a wishful state in general, we would like to merge the stackable-experimental repository with the stackable repository. From there it should be a standard to provide multi-arch images to all products and operators. This means, all merged PR's are available to (currently) both platforms. 
+
+==== Products
+
+Our products have as shown above an overall desireable state. Only thing left to do is to test it in a broughter fashion and merge things into the stackable repository. 
+
+==== Operators
+
+Operators should have a flexible way of setting the repositroy of a product image. Additionally, we would like to have the mulit-arch images for operators build automatically via our CI-pipeline.
```
Comment (maltesander) [inline modules/concepts/pages/road-to-multiarch.adoc:243]:
```suggestion
Operators should have a flexible way of setting the repository of a product image. Additionally, we would like to have the multi-arch images for operators build automatically via our CI-pipeline. 
```

File: modules/concepts/pages/road-to-multiarch.adoc [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+Our products have as shown above an overall desireable state. Only thing left to do is to test it in a broughter fashion and merge things into the stackable repository. 
+
+==== Operators
+
+Operators should have a flexible way of setting the repositroy of a product image. Additionally, we would like to have the mulit-arch images for operators build automatically via our CI-pipeline. 
+
+==== Test Infrastructure
+
+We want to set up a test infrastructure with native ARM-nodes and test both sides of the product and operator images. We doubt currently that it is save to assume that if one side of things is working, the other one is fine too.
```
Comment (maltesander) [inline modules/concepts/pages/road-to-multiarch.adoc:247]:
```suggestion
We want to set up a test infrastructure with native ARM-nodes and test both sides of the product and operator images. We currently doubt that it is save to assume that if one side of things is working, the other one is fine too. 
```

**Comment by Maleware** [comment]
@maltesander I installed a spellchecker and went through your suggestions.

What parts have been hard for you to understand? I know it's been a while, sorry for pulling you back into this 😄

**Comment by fhennig** [comment]
If I've read this correctly, this is describing how the multi-arch images are built. The target for this document is not the platform user: we've already built all the images for them! It is people who want to look behind the scenes, or developers/contributors. I think this _might_ be better as a blogpost? Correct me if I'm wrong! I only skimmed the document

**Comment by Maleware** [comment]
@fhennig  your impression is correct. This document is meant for developer and contributors who want to know more about multi-arch.

We absolutely can wrap it up as a blogpost. I just want to make sure that the knowledge I gathered is stored somewhere accessible, since I wont be the only one working on this in the future (hopefully). 

Where would you put it within the docs? Or is it somewhere completely different?

**Comment by fhennig** [comment]
Yes, I think its great that you wrote it down! We should definitely keep this. I think a blog post is probably the best place for it, but within the docs I think it would fit best under "Contibutors Guide" > "Implementation Guides" (`modules/contributor/pages`).

**Comment by Maleware** [comment]
@vsupalov moved this to Blogpost

**Comment by vsupalov** [comment]
Last polish will be applied in https://github.com/stackabletech/writing/pull/10, and scheduled for publishing over on the blog afterwards.

**Pull request closed without merging**